official/projects/lra/lra_dual_encoder_task.py

```diff
@@ -0,0 +1,319 @@
+# Copyright 2022 The TensorFlow Authors. All Rights Reserved.
```

Comment:
Could we add a unit test to the lra dual encoder task?
Test what's the behavior.
I am wondering if we can subclass the nlp/tasks/dual_encoder.py?

Reply:
Could we add a unit test to the lra dual encoder task? -> Sure, do you have a reference for this? I'll know better what to do

I am wondering if we can subclass the nlp/tasks/dual_encoder.py? -> Probably can't, I adapted from that code but these two are very different

Reply:
No problem